src/components/pages/pricing/tools/tools.tsx

```diff
@@ -34,7 +34,7 @@ const toolBlocks: Tool[] = [
   {
     logo: paintIcon,
     title: 'Taipy Designer',
-    description: 'Simple drag-n-drop constructor.',
+    description: 'The new Python drag-n-drop GUI builder.',
```

Comment:
I don't get the meaning of "Python drag-n-drop builder.". Either it is a Python builder, or it is a drag-n-drop builder. No?

Reply:
From what I understood, it's a GUI builder on top of the Python code.
this aims to be a short description with the essential key words, according to a discussion made with Vincent this morning.

Reply:
If the purpose is to have keywords, why not display simple keywords instead of a description that does not make much sense?

Reply:
Because a succession of words would be weird. People need to understand what is behind this small card the first time they see Taipy Designer

Reply:
So let's find a meaningful sentence. What about:
"A drag-n-Drop GUI builder on top of Python code."?

Reply:
Is it too long?

Reply:
nope, that's perfect

Reply:
changed